File: app/src/main/java/com/grace/placessearch/data/model/VenuesResponse.java
package com.grace.placessearch.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class VenuesResponse {

    @SerializedName("meta")
    @Expose
    private Meta meta;
    @SerializedName("response")
    @Expose
    private VenueListResponse mVenueListResponse;

    public Meta getMeta() {
        return meta;
    }

    public void setMeta(Meta meta) {
        this.meta = meta;
    }

    public VenueListResponse getVenueListResponse() {
        return mVenueListResponse;
    }

    public void setVenueListResponse(VenueListResponse venueListResponse) {
        this.mVenueListResponse = venueListResponse;
    }

}

File: app/src/main/kotlin/com/grace/placessearch/common/data/model/VenuesResponse.kt
package com.grace.placessearch.common.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class VenuesResponse {

    @SerializedName("meta")
    @Expose
    var meta: Meta? = null
    @SerializedName("response")
    @Expose
    var venueListResponse: VenueListResponse? = null

}